content/ko/docs/concepts/services-networking/ingress-controllers.md

```diff
@@ -32,6 +32,7 @@ weight: 40
   Citrix 애플리케이션 딜리버리 컨트롤러에서 작동한다.
 * [Contour](https://projectcontour.io/)는 [Envoy](https://www.envoyproxy.io/) 기반 인그레스 컨트롤러다.
 * [EnRoute](https://getenroute.io/)는 인그레스 컨트롤러로 실행할 수 있는 [Envoy](https://www.envoyproxy.io) 기반 API 게이트웨이다.
+* [Easegress IngressController](https://github.com/megaease/easegress/blob/main/doc/ingresscontroller.md)는 [Easegress](https://megaease.com/easegress/) API 기반 게이트웨이이며, Ingress 컨트롤러로 실행할 수 있습니다.
```

Comment:
Hi @benja-wu 
According to a convention regarding Korean localization, my suggestion is as follows,

```suggestion
* [Easegress 인그레스 컨트롤러](https://github.com/megaease/easegress/blob/main/doc/ingresscontroller.md)는 [Easegress](https://megaease.com/easegress/) API 기반 게이트웨이이며, 인그레스 컨트롤러로서 실행될 수 있다.
```

I hope to note that Korean localization team has own development branch (currently, dev-1.21-ko.7), 
we accept only bug fix PR to main branch in general. But this PR includes some Korean translations. 

Anyway, I think you can 
- continue to proceed in this PR (it is not regular Korean localization contribution process), 
- open an individual PR to dev-1.21-ko.7, or 
- just remove changes in Korean file of this PR. (Korean localization team ourself will follow up English changes within 3 weeks)

Reply:
Thank you for the modification. I will move it into an new individual PR for the Korean doc update.  :-)